astro/src/content/docs/lifecycle/authenticate-users/oauth/scopes.mdx

```diff
@@ -0,0 +1,134 @@
+
+The `Strict` policy uses the requested and consented OAuth scopes to determine which claims to add to the Id token and UserInfo response. The `address`, `email`, `profile`, and `phone` scopes are [provided by FusionAuth](#provided-scopes) and are used to populate claims according to available user data and [Section 5.4 of the OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims).
+
+* `address` - FusionAuth does not include physical addresses in its user data model. A lambda could be used to populate the `address` claim based on custom user data.
+* `email` - populates the `email` and `email_verified` claims
+* `phone` - Populates the `phone_number` claim based on the user's `mobilePhone` value. FusionAuth does not verify phone numbers, so the `phone_number_verified` claim is not included.
+* `profile` - Populates various profile-related data. The following describes how each claim maps to FusionAuth user data.
+
+| Claim | User field |
+| ---- | ---- |
+| `given_name` | `firstName` |
+| `middle_name` | `middleName` |
+| `family_name` | `lastName` |
+| `name` | `fullName` |
+| `preferred_username` | `username` |
+| `birthdate` | `birthDate` |
+| `picture` | `imageUrl` |
+| `locale` | The first value of `preferredLanguages` |
+| `zoneinfo` | `timezone` |
+
+## Provided scopes
+
+FusionAuth provides the following scopes defined by the [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html) specification.
+
+* `address`
+* `email`
+* `phone`
+* `profile`
+
+These scope names cannot be used to define [custom scopes](/docs/get-started/core-concepts/scopes).
+Each of the scopes can be individually `Enabled` and `Required`.
+Provided scopes that have been disabled are considered unknown and will be handled according to the application's <InlineField>Unknown scope policy</InlineField>.
+
+Provided scopes configured as `Required` must be granted user consent during the OAuth [consent prompt](#consent-prompt) in order to successfully complete the OAuth workflow.
+
+### Reserved scopes
+
+FusionAuth reserves the `openid` and `offline_access` scopes.
+These scope names cannot be used to define [custom scopes](/docs/get-started/core-concepts/scopes), and their configuration cannot be changed.
+
+The `openid` scope is used to request identity information according to the [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html) specification.
+
+The `offline_access` scope is used to request a refresh token on the authorization response.
+Its behavior is controlled by the <InlineField>Generate refresh tokens</InlineField> and <InlineField>Enabled grants</InlineField> configurations on the application [OAuth tab](/docs/get-started/core-concepts/applications#oauth).
```

Comment:
What about scopes prefixed with 

idp-link

target-entity:

Are these worth calling out?

Reply:
I added a `Reserved Prefixes` subsection here to highlight the prefixes that can't be used when creating custom scopes.